omisego-sdk/src/main/java/co/omisego/omisego/model/Balance.kt

```diff
@@ -1,23 +1,23 @@
-package co.omisego.omisego.models
-
-import java.math.BigDecimal
-import java.math.RoundingMode
-
+package co.omisego.omisego.model
 
 /**
  * OmiseGO
  *
  * Created by Phuchit Sirimongkolsathien on 11/6/2017 AD.
- * Copyright © 2017 OmiseGO. All rights reserved.
+ * Copyright © 2017-2018 OmiseGO. All rights reserved.
  */
 
+import com.google.gson.annotations.SerializedName
+import java.math.BigDecimal
+import java.math.RoundingMode
+
 /**
  * Represents a balance of a minted token
  *
  * @param mintedToken The minted token corresponding to the balance
  * @param amount The total amount of minted token available for the current user.
  */
-data class Balance(val amount: BigDecimal, val mintedToken: MintedToken) {
+data class Balance(val amount: BigDecimal, @SerializedName("minted_token") val mintedToken: MintedToken) {
```

Comment:
If your backend is always respecting the same case, you can just specify a naming policy on your `Gson` instance, to map the name, to avoid having to specify the `SerializedName` only for those fields.
See https://stackoverflow.com/questions/2370745/convert-json-style-properties-names-to-java-camelcase-names-with-gson

Reply:
Fixed